repo/packages/C/cockroachdb/0/package.json

```diff
@@ -0,0 +1,18 @@
+{
+  "description": "CockroachDB on DC/OS",
```

Comment:
Added a better description. the service guide is https://github.com/dcos/examples/pull/195 and still haven't been merged yet